libgalois/include/katana/GraphTopology.h

```diff
@@ -741,6 +741,10 @@ class KATANA_EXPORT ProjectedTopology : public GraphTopologyTypes {
     return original_to_projected_nodes_mapping_[nid];
   }
 
+  const uint8_t* node_bitmask() const noexcept { return node_bitmask_.data(); }
+
+  const uint8_t* edge_bitmask() const noexcept { return edge_bitmask_.data(); }
```

Comment:
These seem like bad methods to have in the public API. In general I don't think we should expose this kind of thing, but if we need to could we expose it as typed representation like a boolean arrow array or something similar? That way there is no urge for people to directly work on the bytes.

Reply:
Agree with Arthur. 

Return the const arrays here directly. We should NOT be passing around pointers. Arrow requires pointers, so we can call `.data()` when we call Arrow but we don't need to deal with pointers in our code. 

Please add doxygen comments to the functions saying what set and unset in the mask represents. 

Also maybe add `projected_` prefix to the names?

Reply:
I have changed this now by using arrow's Bitmap class